parachains/runtimes/assets/asset-hub-polkadot/src/constants.rs

```diff
@@ -74,7 +74,7 @@ pub mod fee {
 		type Balance = Balance;
```

Comment:
FYI this stuff can easily be rfactored out to `runtime_common` with something like `RefTimeToFee<const COEFFICIENT>` or similar.